integration-tests/Cargo.toml

```diff
@@ -184,3 +184,8 @@ std = [
     "hydradx-runtime/std",
     "pallet-staking/std",
 ]
+
+# we don't include integration tests when benchmarking feature is enabled
+runtime-benchmarks = [
```

Comment:
can you please write a brief summary of what was the main change about these feature flags? I mean, how we intend to use them from now on. Thanks!